### DISCUSSION TITLE
stream private

### DISCUSSION THREAD

**Discussion by fsan2val** [owncast/owncast#3758] [Q&A] [answered]
Hi guys.

A short question that I couldn't find. Can the stream be set to private? If it is positive, how does it work? Because I can't find options in the panel

Thank you and happy day

**Comment by gabek** [comment, marked as answer]
See https://github.com/owncast/owncast/discussions/649 and https://github.com/owncast/owncast/issues/489 for previous discussions.

**Reply by fsan2val** [reply]
Thanks for your response, I'll check it out. thanks greetings
